config/olm/kubernetes/0.1.0/bundle.Dockerfile

```diff
@@ -9,7 +9,5 @@ LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.3.0
```

Comment:
the latest operator sdk is 1.13.0, please double check that this value is correct